ims/ostis_tech/semantic_network_represent/section_scs_code/section_scs_code_scsi/scs_code_htmls/scs_code_core.html

```diff
@@ -7,7 +7,7 @@
 Т.е. имя <i><sc_element sys_idtf="sc_element">sc-элемента</sc_element></i>, из которого выходит указанная <i><sc_element sys_idtf="sc_arc">дуга</sc_element></i>, указывается первым в <i><sc_element sys_idtf="triple_set">тройке</sc_element></i>, а имя <i><sc_element sys_idtf="sc_element">sc-элемента</sc_element></i>, в который входит эта <i><sc_element sys_idtf="sc_arc">дуга</sc_element></i>, указывается третьим в <i><sc_element sys_idtf="triple_set">тройке</sc_element></i>.<br>
 Для указания типов <i><sc_element sys_idtf="sc_element">sc-элементов</sc_element></i> могут быть использованы следующие правила уточнения типа <i><sc_element sys_idtf="sc_element">sc-элемента</sc_element></i>:<br>
 &lt;&quot; <b>имя элемента с уточнением типа</b> &quot;&gt; ::= &lt;&quot; <b>тип элемента</b> &quot;&gt; # [&quot;&lt;&quot; <b>имя элемента</b> &quot;&gt;&quot;]<br>
-&lt;&quot; <b>тип элемента</b> &quot;&gt; ::= (&quot; sc_node || sc_link || sc_arc_common || sc_edge || sc_arc_main || sc_arc_access &quot;)<br>
+&lt;&quot; <b>тип элемента</b> &quot;&gt; ::= (&quot; sc_node || sc_node_link || sc_common_arc || sc_common_edge || sc_main_arc || sc_membership_arc &quot;)<br>
```

Comment:
if you decided to change sc_link, sc_arc_common and others then change them everywhere